src/EnergyPlus/HVACControllers.cc

```diff
@@ -3555,6 +3520,47 @@ Label100: ;
 	}
 
 	void
+	GetControllerNameAndIndex(
+		int const WaterInletNodeNum, // input actuator node number
+		std::string & ControllerName, // controller name used by water coil
+		int & ControllerIndex, // controller index used by water coil
+		bool & ErrorsFound // true if matching actuator node not found
+	)
+	{
+
+		// SUBROUTINE INFORMATION:
+		//       AUTHOR         Richard Raustad
+		//       DATE WRITTEN   June 2017
+
+		// PURPOSE OF THIS FUNCTION:
+		// This subroutine checks that the water inlet node number is matched by
+		// the actuator node number of some water coil and passed back controller name and index
+
+		// FUNCTION LOCAL VARIABLE DECLARATIONS:
+		int ControlNum;
```

Comment:
Could just define this inside the `for` loop below; no big deal for now though.

Reply:
These old habits are hard to break, but slowly getting there. I do get a little confused when a loop variable is used multiple times and inside IF blocks where to initialize or to initialize multiple times.